source/index-rw.html.md

```diff
@@ -24,6 +24,8 @@ includes:
   - api/graph
   - api/areas
   - api/webshot
+  - api/topic
```

Comment:
if we're making these services part of the core, then you may also need to update the other index files for gfw prep and the default one - you may want to reach out to the respective PMs on this, or check with @pablopareja